lab1/Pasichnichenko_Task1.py

```diff
@@ -0,0 +1,12 @@
+# 3.1
+print("'Multi-paradigm programming languages':Task 1")
+print("Pasichnichenko Illia IKM-221a")
+
+# 3.2
+for i in range(45):
+    separator = ', ' if i < 44 else ''
```

Comment:
Wonderful finding! ternary operator is awesome.

Reply:
tnx :)